apps/website/src/pages/docs/avatar.mdx

```diff
@@ -13,73 +13,53 @@ import * as AllExamples from 'examples';
+The avatar is an essential UI component, making it easier to identify a person or organization by their photo, icon, or initials.
 
-## Appearance
+## Usage
+
+Different properties can be passed to the `<Avatar>` component to customize it according to the user's needs.
+
+- `title` prop can be used to specify the name or title associated with the avatar.
+- `abbreviation` prop allows users to provide a short abbreviation or initials for the avatar.
+- `image` prop can be passed to set a custom image source or icon that will be displayed within the avatar.
 
 ### Sizes
 
-There are four sizes of avatars:
+Avatars can be displayed in various sizes, including `small`, `medium`, `large`, and `x-large`. These options offer users the flexibility to choose the most appropriate avatar size.
 
 <LiveExample src='Avatar.sizes.tsx'>
   <AllExamples.AvatarSizesExample client:load />
 </LiveExample>
 
-- `small`
-- `medium`
-- `large`
-- `x-large`
-
-Which size to use is at your discretion. Determine which size is best for your context based on available space.
-
-### Initials
-
-If a user does not upload a photo, the user’s first & last initials are displayed. You can display a maximum of 2 letters and a minimum of 1 letter. In the off chance there are user accounts without any name associated, you can display the first letter of the user’s email address.
-
-<LiveExample src='Avatar.initials.tsx'>
-  <AllExamples.AvatarInitialsExample client:load />
-</LiveExample>
-
-### Icon
-
-An icon may be display instead of initials, typically used by organizations instead of a person.
-
-<LiveExample src='Avatar.icon.tsx'>
-  <AllExamples.AvatarIconExample client:load />
-</LiveExample>
+By making use of the `size` prop and selecting one of the predefined values mentioned above, you can easily set the size of the `Avatar` component.
 
-### Picture
+### Status indicators
 
-If a user uploads a photo we display their photo instead of their initials.
+Status indicators are indicators that display in the bottom right of the avatar. Statuses can be set using the `status` prop. The available statuses are: `online`, `away`, `busy` and `offline`.
 
-<LiveExample src='Avatar.picture.tsx'>
-  <AllExamples.AvatarPictureExample client:load />
+<LiveExample src='Avatar.statuses.tsx'>
+  <AllExamples.AvatarStatusesExample client:load />
 </LiveExample>
 
-### Status indicators
+We suggest not using statuses on the small size avatar.
 
-You may want to indicate the online status of a user. Status indicators display in the bottom right of the avatar. We suggest not using statuses on the small size avatar. The available statuses are:
+## Variants
 
-- `online`
-- `away`
-- `busy`
-- `offline`
+An avatar can be customized to show initials, icons, and pictures.
 
-<LiveExample src='Avatar.statuses.tsx'>
-  <AllExamples.AvatarStatusesExample client:load />
+<LiveExample src='Avatar.main.tsx'>
```

Comment:
Before merging, I think the example file names can be renamed to match the new titles.